src/main/java/frc/robot/RobotContainer.java

```diff
@@ -116,6 +120,10 @@ public RobotContainer() {
     configureBindings();
 
     drive.setDefaultCommand(driveRobotOriented);
+    autoChooser = new SendableChooser<>();
+    SmartDashboard.putData("choose auto", autoChooser);
+    /* This is where you put auto commands. */
```

Comment:
Move this stuff into a separate method, so that we have an organized place to add stuff to the auto chooser. Call it configureAutos() or something.